remotespark/livyclientlib/pandaslivyclientbase.py

```
@@ -31,7 +31,8 @@ def _execute_dataframe_helper(self, context_name, command):
             return records_text
 
     def get_columns(self, context_name, command):
-        return self.execute(self.make_context_columns(context_name, command))
+        return str(self.execute(self.make_context_columns(context_name,
```

Yes, we need to propagate the exception here.